security/pkg/pki/util/utils.go

```diff
@@ -0,0 +1,60 @@
+// Copyright 2017 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package util
```

Comment:
I suggest to move this function into `https://github.com/istio/istio/blob/master/security/pkg/pki/ca/controller/secret.go`.
`util`, `common` are not suggested package names and we should avoid conflating them when possible, https://blog.golang.org/package-names

Reply:
@incfly Moving to the file `security/pkg/pki/ca/controller/secret.go` causes the problem `import cycle`. I currently placed it to `security/pkg/pki/ca/ca.go`. Any other thoughts on this?